external/extraMbedtls/generate-pkcs7.c

```diff
@@ -788,6 +792,8 @@ int to_pkcs7_already_signed_data(unsigned char **pkcs7, size_t *pkcs7Size, const
 	sigs = calloc(1, sizeof(char*) * keyPairs);
 	sig_sizes = calloc(1, sizeof(size_t) * keyPairs);
 	if (!sigs || !sig_sizes) {
+		if (sigs) free(sigs);
+		if (sig_sizes) free(sig_sizes);
```

Comment:
You already have clean-up code at the end of the function, you can reuse that by setting another label and jumping to that. Not a big deal to fix now, but something to consider in the future. It's best not to repeat yourself too much on clean ups, or you can miss one in a refactor.